rust/lang/security/audit/ldap-query.yml

```diff
@@ -0,0 +1,18 @@
+rules:
+  - id: ldap-injection
+    patterns:
+      - pattern-either:
+          - pattern: |
+              let $DATA = requests::$METHOD(...).unwrap();
+              ...
+              let mut $LDAP = LdapConn::new(...)?;
+              ...
+              let $RESULT = $LDAP.search(...,$DATA,...)?.success()?;
+
```

Comment:
This looks like it should be a taint mode rule! https://semgrep.dev/docs/writing-rules/data-flow/taint-mode/